src/openapi/streaming/connection/transport/websocket-transport.js

```diff
@@ -135,9 +157,17 @@ function parseMessage(rawData) {
             const payloadBuffer = new Uint8Array(
                 rawData.slice(index, index + payloadSize),
             );
-            data = String.fromCharCode.apply(null, payloadBuffer);
-            data = decodeURIComponent(escape(data));

```

Comment:
@lukeapage Do we really need to sanitise the data
escape is deprecated now
I have discussed it a bit with @michax and removed

Reply:
As discussed on slack. Can be removed